app/controllers/application_controller.rb

```diff
@@ -169,11 +169,11 @@ def instrument_with_timer
   # given cache_key and fills out custom response headers.
   # If the attempt results in a cache miss, then the response is generated normally and
   # will be stored in the cache.
-  def fetch_from_or_store_in_cache(skip_cache, cache_key, httpdate, event_name)
+  def fetch_from_or_store_in_cache(skip_cache, cache_key, httpdate, _event_name)
     if skip_cache
       yield
     else
-      MetricUtil.log_analytics_event(event_name + "_cache-requested", current_user) unless skip_cache
+      MetricUtil.put_metric_now("samples.cache.requested", 1) unless skip_cache
```

Comment:
Is there a meaning behind logging the value `1`?

Reply:
`put_metric_now` expects some `value` as its second parameter. Seems like other calls to the function use it to log extra info such as runtime, number of samples created, etc. I'm not actually sure what the history behind the `1` for caching metrics is, since this is carried over from the logging statement from the older version of the report page (I assume it's some placeholder default value?).

@gregdingle Do you have any more context on this?

Reply:
Might it mean "1 occurrence" for downstream aggregation/summation later?

Reply:
Actually that is a good question but I believe Charles is right. We could eventually report more samples requested and do a sum on datadog side.